consensus/client/src/input.rs

```diff
@@ -120,7 +120,7 @@ impl TransactionInput {
 
     #[wasm_bindgen(setter = signatureScript)]
     pub fn set_signature_script_from_js_value(&mut self, js_value: JsValue) {
-        self.inner().signature_script = js_value.try_as_vec_u8().expect("invalid signature script");
+        self.set_signature_script(js_value.try_as_vec_u8().expect("invalid signature script"));
```

Comment:
Make this return `Error` instead of panicking.  (all functions should always return errors unless there is some critical condition that doesn't allow user to continue)